portfoliostylesheet.css

```diff
@@ -0,0 +1,332 @@
+* {
+	box-sizing:border-box;
+}
+
+a {
+    text-decoration: none;
+}
+a:link, a:visited {
+    color: #B4535A;
+}
+
+a:hover {
+    color: #D0D2CA;
+}
+
+html { 
+	font-size:16px;
+	font-family:"verdana", sans-serif;
+}
+
+body {
+	margin:0px;
+	padding:0px;
+}
+	
+h3 {
+	font-family:"avenir book";
+	text-size:1rem;
+	color:#B28537;
+}
+
+.splitGrid {
+	display: grid;
+	grid-template-columns: 4fr 1fr ;
+	margin-left: 15%;
+	margin-right:15%;
+}
+
+
+.footer {
+	margin-left:40%;
+	font-size:.6rem;
+	border-top:5px;
+}
+
+
+
+/*Elements from the right-hand side navigation bar*/
+
+.name {
+	font-family:"Avenir Book";
+	font-weight:"bold";
+	padding-top:15px;
+	display:flex;
+	align-items:center;
+	text-size:1.4rem;
+}
+
+
+.contactBar {
+	background-color: #C6D2D1;
+	display: flex;
+	flex-direction: column;
+	flex-wrap:wrap;
+	align-items:left;
+	justify-content:flex-start;
+}
+
+.header {
+	background-color:#215097;
+	display: flex;
+	align-items:center;
+	flex-direction: column;
+	flex-wrap:nowrap;
+	width: 100%;
+	color:#D0D2CA;
+	padding-top:40px;
+	padding-bottom:5px;
+}
+
+.profilePic {
+	border-radius: 50%;
+	width: 80%;
+	display: flex;
+	align-items:center;
+	margin:0 auto;
+}
+
+.status {
+	font-family:"avenir book";
+/*	font-style:"medium";*/
+	font-size:.8rem;
+	padding-bottom:15px;
+	padding-top:15px;
+}
+
+.contactDeets {
+	display: flex;
+	flex-direction:column;
+	align-items:left;
+/*	padding-left:25px;*/
+	padding-right:25px;
+	margin-left:15%;
+	margin-right:5px;
+	padding-top:40px;
+	line-height:130%;
+}
+
+.contactIcon{
+	color:blue;
+}
+
+.educationDeets {
+	display: flex;
+	flex-direction:column;
+	align-items:left;
+/*	padding-left:25px;*/
+	padding-right:25px;
+	margin-left:15%;
+	margin-right:5px;
+	wrap:wrap;
+	padding-top:20px;
+	line-spacing:120%;
+}
+
+.langSkills {
+	display:flex;
+	flex-direction:column;
+	align-items:left;
+/*	padding-left:25px;*/
+	padding-right:25px;
+	margin-left:15%;
+	margin-right:5px;
```

Comment:
This is some highly specific maths using both margins and padding and different units of measurement - do you think there might be a simpler way of achieving the same thing?

Also, a lot of the code for this class is also used for some other classes. That means that if you change it here, you have to go and change it in other places too. Instead, put all the common css together like so:
```css
.contactDeets, .educationDeets, .langSkills {
   /* common code */
}

Reply:
And for the code that's different just use separate selectors like before